src/indexers/governance-proposals-indexer.ts

```diff
@@ -0,0 +1,251 @@
+import {
+  CallDisputedEvent,
+  CallDisputedResolvedEvent,
+  CallExecutedEvent,
+  CancelledEvent,
+} from '@badger-dao/sdk/lib/contracts/TimelockController';
+import { GovernanceProxyMock } from '@badger-dao/sdk/lib/governance/mocks/governance.proxy.mock';
+
+import { getDataMapper } from '../aws/dynamodb.utils';
+import { GovernanceProposals } from '../aws/models/governance-proposals.model';
+import { GovernanceProposalsDisputes } from '../aws/models/governance-proposals-disputes.interface';
+import { GovernanceProposalsStatuses } from '../aws/models/governance-proposals-statuses.interface';
+import { getSupportedChains } from '../chains/chains.utils';
+import { getLastProposalUpdateBlock } from '../governance/governance.utils';
+
+export async function updateGovernanceProposals() {
+  console.info('Updating governance proposals started');
+
+  const mapper = getDataMapper();
+
+  for (const chain of getSupportedChains()) {
+    const timelockAddress = chain.sdk.governance.timelockAddress;
```

Comment:
this 1:1 mapping of chain to timelock addr makes having a test addr not available

Reply:
this is till we will add timelocks to the registry and sdk

Reply:
i believe its about mapping on sdk, and mrbasado wants to save possibility of having test addr